src/pfs/pfs.proto

```diff
@@ -245,6 +245,7 @@ message ProjectInfo {
   AuthInfo auth_info = 3;
   google.protobuf.Timestamp created_at = 4;
   map<string, string> metadata = 5;
+  string created_by = 6;
```

Comment:
we should consider as a part of this ticket whether we should make changes in the CLI. Should CreatedBy be presented in any of the commands?